dnf5-plugins/builddep_plugin/builddep.hpp

```diff
@@ -57,6 +57,8 @@ class BuildDepCommand : public Command {
     std::vector<std::string> srpm_file_paths{};
     std::vector<std::pair<std::string, std::string>> rpm_macros{};
 
+    std::string spec_type{"auto"};
```

Comment:
I prefer to use a "real" enumeration instead of strings.

For example: `enum class ArgType {AUTO, SPEC, SRPM} arg_type{ArgType::AUTO};`
And then using `switch` for these 3 cases instead of `if`.

Reply:
Thank you! Changed to an enum class.